Casks/imhex.rb

```diff
@@ -0,0 +1,23 @@
+cask "imhex" do
+  version "1.19.3"
+  sha256 "2bce4b8e5617b91bf771ed58654b9fcd88f9a195d63e83eb82fdc6514d97610c"
+
+  url "https://github.com/WerWolv/ImHex/releases/download/v#{version}/imhex-#{version}-Darwin.dmg",
+      verified: "github.com/WerWolv/ImHex"
```

Comment:
```suggestion
      verified: "github.com/WerWolv/ImHex/"
```
Not strictly necessary, I supposed, but to be consistent with the overwhelming majority of the codebase.

Reply:
Was added by `brew style --fix <cask>` either ways.

Reply:
I don't see it in your latest commit, though.

Reply:
That's weird, because in my [repo](https://github.com/jehoffmann/homebrew-cask/commit/21fcec6d0cf278a802c186f2886daeadaa54b09c) there is the / at the end.